app/src/main/java/com/brainwallet/data/model/IntroLanguageResource.kt

```diff
@@ -133,14 +133,6 @@ class IntroLanguageResource {
             "Sei sicuro di voler cambiare la lingua in italiano?",
             Language.ITALIAN
             ),
-            IntroLanguage(
-                Language.BLANK.code,
-                Language.BLANK.title,
-                " ",
-                R.raw.english,
```

Comment:
got it we are using raw resource, that's why previously I propose to remove the asset pack

Reply:
Oh.  This was both to learn about PAD...We want to keep the .aab smaller.  
I would rather remove the raw assets....will change